devbox.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packages": [
+    "bash@latest",
+    "clusterctl@1.6.2",
+    "envsubst@1.4.2",
+    "gnumake@4.4.1",
+    "ginkgo@2.17.0",
+    "go@1.22.1",
```

Comment:
1.22.3?

Reply:
```
direnv: loading ~/go/src/github.com/nutanix-cloud-native/cloud-provider-nutanix/.envrc
direnv: using devbox
Info: Ensuring packages are installed.Error: go@1.22.3: package not found
```

Reply:
Switched to `1.22.2` for the time being.